### DISCUSSION TITLE
Prognosis for npm@8?

### DISCUSSION THREAD

**Discussion by richb-hanover** [npm/feedback#494] [CLI] [answered]
I checked out the [Public Roadmap](https://github.com/npm/roadmap/projects/1) and wonder when we might expect npm@8 to hit the streets. (I am [following a bug](https://github.com/npm/cli/issues/3525#issuecomment-878290633) that would be nice to tie up, but seems held up until v8...) Thanks.

**Comment by MylesBorins** [comment]
pinging @darcyclarke to chime in

**Comment by darcyclarke** [comment, marked as answer]
@richb-hanover can't give any specific dates/timelines at the moment as we're still fleshing out what changes would be made in v8 beyond major engines/dependency updates; That said, an educated/best guess at this moment would be, at the very earliest, late this year but more likely early/mid 2022 (again, that's speculative). We'll make public a set of the breaking changes we expect for that release as soon as it's ready.

**Reply by richb-hanover** [reply]
@darcyclarke  Thank you for this comprehensive response.  It is regrettable that so much depends on a version of node-gyp that still relies on request() and , but I am sure you've thought hard about that dependency... Best regards,  Rich
